docs/tasks/debug-application-cluster/audit.md

```diff
@@ -72,6 +72,9 @@ In both cases, audit events structure is defined by the API in the
 `audit.k8s.io` API group. The current version of the API is
 [`v1beta1`][auditing-api].
 
+**Note:** in case of patches, request objects are arrays, containing operations, not the appropriate
+API objects. You should design your system to behave correctly in this case.
```

Comment:
Drop "You should design your system to behave correctly in this case." It's already implied.

Reply:
Done